pkgs/development/libraries/gstreamer/rs/default.nix

```
@@ -114,7 +115,7 @@ let
       "reqwest" # tests hang on darwin
       "threadshare" # tests cannot bind to localhost on darwin
       "webp" # not supported on darwin (upstream crate issue)
-    ] ++ lib.optionals (!gst-plugins-base.glEnabled) [
+    ] ++ lib.optionals (!gst-plugins-base.glEnabled || withGtkPlugins) [
```

I think this is a copy-paste error.
```suggestion
    ] ++ lib.optionals (!gst-plugins-base.glEnabled || !withGtkPlugins) [
```